docs/source/en/api/models/unet.mdx

```diff
@@ -0,0 +1,37 @@
+# UNet
+
+The [UNet](https://huggingface.co/papers/1505.04597) model was originally introduced by Ronneberger et al for biomedical image segmentation, but it is also commonly used in 🤗 Diffusers because it outputs images that are the same size as the input. There are several variants of the UNet model in 🤗 Diffusers, depending on it's number of dimensions and whether it is a conditional model or not.
```

Comment:
Nice to see HF papers here \m/

Reply:
Might be nice to include that the UNet is one of the most important components of a diffusion system as it's utilized to facilitate the actual diffusion process.